docker/puppetdb/docker-entrypoint.sh

```diff
@@ -11,10 +11,10 @@ if [ ! -d "/etc/puppetlabs/puppetdb/ssl" ] && [ "$USE_PUPPETSERVER" = true ]; th
     sleep 1
   done
   set -e
-  /opt/puppetlabs/bin/puppet config set certname "$HOSTNAME"
-  /opt/puppetlabs/bin/puppet config set server "$PUPPETSERVER_HOSTNAME"
-  /opt/puppetlabs/bin/puppet agent --verbose --onetime --no-daemonize --waitforcert 120
-  /opt/puppetlabs/server/bin/puppetdb ssl-setup -f
+  /ssl.sh
+  /ssl-setup.sh -f
```

Comment:
@puppetlabs/puppetdb I had to copy/paste/modify the current ssl-setup.erb file so it didn't require EZBake and could run under /bin/sh (alpine doesn't have bash).

This ssl-setup.sh script is specific to puppetdb so would remain here, and then we'd need to remember it when we make future changes to the ssl-setup.erb file.

We could probably refactor the ssl-setup.erb from bash to sh (I did that below) but not sure how to address the EZBake lines.